skyline-vscode/react-ui/src/App.js

```diff
@@ -252,66 +188,7 @@ function App() {
-                        <Subheader icon="database">Peak Memory Usage</Subheader>
-                            <Row>
-                                <Col>
-                                <Badge className="float-end" bg="secondary">{analysisState["hardware_info"]["gpus"][0]}</Badge>
-                                <Badge className="float-end" bg="success">Local</Badge>
-                                </Col>
-                            </Row>
-                        <div className="innpv-subpanel-content">
-                            <BarSlider
-                                percentage={sliderMemory[0]}
-                                limitPercentage='100'
-                                handleResize={onMemoryResize}
-                                style={{ height: '100%' }}
-                            />
-
-                            <div className="innpv-subpanel-sidecontent">
-                                <NumericDisplay
-                                    top="Peak Usage"
-                                    number={sliderMemory[1]}
-                                    bottom="Megabytes"
-                                />
-                                <div className="innpv-separator" />
-                                <NumericDisplay
-                                    top="Maximum Capacity"
-                                    number={sliderMemory[2]}
-                                    bottom="Megabytes"
-                                />
-                            </div>
-                        </div>
-                    </div>
-                    <div className="innpv-memory innpv-subpanel">
-                        <Subheader icon="database">Throughput</Subheader>
-                        <div className="innpv-subpanel-content">
-                            <BarSlider
-                                percentage={sliderThroughput[0]}
-                                limitPercentage='100'
-                                handleResize={onThroughputResize}
-                                style={{ height: '100%' }}
-                            />
-
-                            <div className="innpv-subpanel-sidecontent">
-                                <NumericDisplay
-                                    top="Throughput"
-                                    number={sliderThroughput[1]}
-                                    bottom="samples/second"
-                                />
-                                <div className="innpv-separator" />
-                                <NumericDisplay
-                                    top="Predicted Maximum"
-                                    number={sliderThroughput[2]}
-                                    bottom="samples/second"
-                                />
-                            </div>
-                        </div>
-                    </div>
+                    <MemThroughputContainer analysisState={analysisState} SENDMOCK={SENDMOCK} />
```

Comment:
I would change the initial SENDMOCK to sendMock so we are not confused on the variable name

Reply:
Changed variable to sendMock